### DISCUSSION TITLE
Custom PCB Implementation

### DISCUSSION THREAD

**Discussion by elgranshaman** [Bodmer/TFT_eSPI#1332] [Q&A - General]
Hello all,

I want to build a custom PCB using an LCD TFT panel instead of a module. As I understand, for this library to work, the SPIs and other control signals need to be converted to 16-bit parallel using the 74CH4xxx IC's.

I found at Waveshare, [the schematic](https://www.waveshare.com/w/upload/7/71/3.5inch_TFT_Touch_Shield_Schematic.pdf) for this implementation.

I am planning to get a 3.5" TFT LCD with a backlight and an ILI9486 or ILI9488 driver. Are they the same? They look like it from the outside at Ilitek website.

On my custom PCB, I will have an ESP32 plus the SPI to 16-bit circuit without level conversion as it is already 3.3V and nothing the touchscreen and SD functionality.

Can you please confirm that all my assumptions are correct and I can use this library over the above-mentioned configuration?

Thank you

**Comment by elgerg** [comment]
Have you seen these WT32-SC01:
https://www.aliexpress.com/item/1005002244680082.html?spm=a2g0s.9042311.0.0.27424c4dF2CWmo
I have one and just ordered another. They are quite nice.

**Reply by elgranshaman** [reply]
Thanks for sharing! But need to design a custom PCB :)
